### PR TITLE
Make 'change log' a single word

### PR DESCRIPTION
**Pull request by mesaugat**
This PR basically replaces `change log` with `changelog` across all files.

There were inconsistencies in various files regarding the use of `change log` and `changelog`. I think `changelog` is the correct way to write it.

### CODE REVIEW

File: spec/unit/octo_fetcher_spec.rb [hunk 1]
```diff
@@ -265,7 +265,7 @@
                             "diff_url"  => "https://github.com/skywinder/changelog_test/pull/21.diff",
                             "patch_url" => "https://github.com/skywinder/changelog_test/pull/21.patch" },
                         "body"           =>
-                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in change log with #20" }
+                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in changelog with #20" }
```
Comment (olleolleolle) [inline spec/unit/octo_fetcher_spec.rb]: Undo the changes in this file.

They refer to data in another repository, the changing of which is outside the scope of this find-and-replace PR.

File: spec/unit/octo_fetcher_spec.rb [hunk 2]
```diff
@@ -265,7 +265,7 @@
                             "diff_url"  => "https://github.com/skywinder/changelog_test/pull/21.diff",
                             "patch_url" => "https://github.com/skywinder/changelog_test/pull/21.patch" },
                         "body"           =>
-                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in changelog with #20" }
+                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in change log with #20" }
```
Comment (olleolleolle) [inline spec/unit/octo_fetcher_spec.rb]: The "title" (in this document, too) 

```
"title" => "Merged br (should appear in changelog with  #20)",
```

needs to go back to what it was before.

Comment (mesaugat) [reply spec/unit/octo_fetcher_spec.rb]: Yes, just saw this.
